sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -85,126 +85,122 @@ client = FormRecognizerClient(endpoint, credential)
+table data, from forms of the type you trained your models on. The output of models trained with and without labels
 differs as described below.
 
 #### Using models trained without labels
-Models trained without labels consider each form page to be a different form type.  For example, if you train your 
-model on 3-page forms, it will learn that these are three different types of forms.  When you send a form to it for 
+Models trained without labels consider each form page to be a different form type. For example, if you train your
+model on 3-page forms, it will learn that these are three different types of forms. When you send a form to it for
 analysis, it will return a collection of three pages, where each page contains the field names, values, and locations,
 as well as table data, found on that page.
 
 #### Using models trained with labels
-Models trained with labels consider a form as a single unit.  For example, if you train your model on 3-page forms 
-with labels, it will learn to extract field values from the locations you've labeled across all pages in the form. 
-If you sent a document containing two forms to it for analysis, it would return a collection of two forms, 
+Models trained with labels consider a form as a single unit. For example, if you train your model on 3-page forms
+with labels, it will learn to recognize field values from the locations you've labeled across all pages in the form.
+If you sent a document containing two forms to it for analysis, it would return a collection of two forms,
 where each form contains the field names, values, and locations, as well as table data, found in that form.
 Fields and tables have page numbers to identify the pages where they were found.
 
 ### Managing Custom Models
-Using the `CustomFormClient`, you can get, list, and delete the custom models you've trained. 
-You can also view the count of models you've trained and the maximum number of models your subscription will 
+Using the `FormTrainingClient`, you can get, list, and delete the custom models you've trained.
+You can also view the count of models you've trained and the maximum number of models your subscription will
 allow you to store.
 
 ## Examples
 
 The following section provides several code snippets covering some of the most common Form Recognizer tasks, including:
 
-* [Extract Receipts](#extract-receipts "Extract receipts")
-* [Extract Form Layouts](#extract-form-layouts "Extract form layouts")
+* [Recognize Receipts](#recognize-receipts "Recognize receipts")
+* [Recognize Content](#recognize-content "Recognize Content")
+* [Recognize Forms Using a Custom Model](#recognize-forms-using-a-custom-model "Recognize Forms Using a Custom Model")
 * [Train a Model](#train-a-model "Train a model")
-* [Extract Forms Using a Model](#extract-forms-using-a-model "Extract forms using a model")
+* [Manage Your Models](#manage-your-models "Manage Your Models")
```

Comment:
We'll ask Cala, but I'm thinking that Manage Your Models might not be a champion scenario